monifu/shared/src/main/scala/monifu/reactive/internals/operators/debounce.scala

```diff
@@ -45,7 +45,7 @@ private[reactive] object debounce {
         private[this] var hasValue = false
 
         locally {
-          scheduleNext(timeoutMillis)
+          scheduleNext(initialDelay.toMillis)
```

Comment:
Let me see if I got this. Basically an `initialDelay` doesn't make sense on the debounce operator, but if we don't want to wait the `timeout` to have the Observable emitting its items then we should basically have a value that is relative to the _last event emitted_. What I don't understand is if we don't have